Neos.Flow/Tests/Functional/Reflection/ReflectionServiceTest.php

```diff
@@ -356,12 +356,57 @@ public function annotatedArrayTypeHintsWorkCorrectly()
      */
     public function unionReturnTypesWorkCorrectly()
     {
-        $returnTypeA = $this->reflectionService->getMethodDeclaredReturnType(Fixtures\DummyClassWithUnionTypeHints::class, 'methodWithUnionReturnTypeA');
-        $returnTypeB = $this->reflectionService->getMethodDeclaredReturnType(Fixtures\DummyClassWithUnionTypeHints::class, 'methodWithUnionReturnTypesB');
-        $returnTypeC = $this->reflectionService->getMethodDeclaredReturnType(Fixtures\DummyClassWithUnionTypeHints::class, 'methodWithUnionReturnTypesC');
+        $returnTypes = [
+            'returnTypeA' => $this->reflectionService->getMethodDeclaredReturnType(Fixtures\DummyClassWithUnionTypeHints::class, 'methodWithUnionReturnTypeA'),
+            'returnTypeB' => $this->reflectionService->getMethodDeclaredReturnType(Fixtures\DummyClassWithUnionTypeHints::class, 'methodWithUnionReturnTypesB'),
+            'returnTypeC' => $this->reflectionService->getMethodDeclaredReturnType(Fixtures\DummyClassWithUnionTypeHints::class, 'methodWithUnionReturnTypesC'),
+        ];
+
+        self::assertEquals(
+            [
+                'returnTypeA' => 'string|false',
+                'returnTypeB' => '\Neos\Flow\Tests\Functional\Reflection\Fixtures\DummyClassWithUnionTypeHints|false',
```

Comment:
In PHP FQCN in strings never have a leading backslash – should we handle that the same way? At least consistency would be great, seeing the next test expects those without a leading backslash…

Reply:
The tests just took what the type hints created for granted. May aswell be buggy.